pos_partner_no_create/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 - 2020 BEES coop SCRLfs
+#   - Elouan Lebars <elouan@coopiteasy.be>
+#   - Rémy Taymans <remy@coopiteasy.be>
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+#   - Elise Dupont
+#   - Thibault François
+#   - Grégoire Leeuwerck
+#   - Houssine Bakkali <houssine@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
all that people for that module ? ;-)

Reply:
Yes it was a great collective effort ^^. I am kidding, it's a split from a big module, hence all the names. I remove them.